.gitignore

```diff
@@ -14,3 +14,4 @@ nbproject
 GTAGS
 GRTAGS
 GPATH
+vssver2.scc
```

Comment:
@tius2000 Please put a comment line above this one saying "Microsoft Visual SourceSafe files". Ex:

```
# Microsoft Visual SourceSafe files
vssver2.scc
```

Reply:
It will be great if you can add also comments for the other type of files. For example Eclipse IDE files, NetBeans IDE files, etc.

Reply:
I'm not sure about the other file types @slaff, so I just added a comment line for Visual SourceSafe.

Reply:
Great! Thanks!